{{cookiecutter.profile_name}}/lsf_submit.py

```diff
@@ -65,7 +65,7 @@ def cluster(self) -> dict:
 
     @property
     def threads(self) -> int:
-        return self.job_properties.get("threads", CookieCutter.get_default_threads())
+        return self.job_properties.get("threads", 1)
```

Comment:
This could be slightly simplified to `self.job_properties.get("threads")`

Reply:
But it would return `None` if threads wasn't in job properties right? I know that's an edgecase that probably doesn't happen - but still I'm paranoid haha.